install_latest_gauge_nightly.ps1

```diff
@@ -7,12 +7,15 @@ else {
     $bit="x86"
 }
 
-$gauge_file_name="gauge-$($gauge_latest_nightly_version)-windows.$($bit).exe"
+$gauge_exe_folder_name="gaugeExe"
+$gauge_file_name="$gauge_exe_folder_name\\gauge.exe"
+$gauge_zip_name="gauge-$($gauge_latest_nightly_version)-windows.$($bit).zip"
 
-Invoke-WebRequest -Uri "https://bintray.com/gauge/Gauge/download_file?file_path=windows%2F$gauge_file_name" -OutFile $gauge_file_name
+Invoke-WebRequest -Uri "https://bintray.com/gauge/Gauge/download_file?file_path=windows%2F$gauge_zip_name" -OutFile $gauge_zip_name
+Expand-Archive $gauge_zip_name -DestinationPath $gauge_exe_folder_name
```

Comment:
The destination folder needs to be added to `PATH` else it wont be available to subsequent steps.